### PR TITLE
Update attaching-files.rst

### PR DESCRIPTION
**Pull request by amyblais**
I will continue to work on adding benefits and best practices.

### CODE REVIEW

**Comment by jasonblais** [comment]
@JeffSchering I think this might still be WIP

File: source/help/messaging/attaching-files.rst [hunk 1]
```diff
@@ -3,6 +3,9 @@ Attaching Files
 
 --------------
 
+Sharing project plans, new designs, and images can make your messages more
```
Comment (it33) [inline source/help/messaging/attaching-files.rst]: maybe emphasize photos, video, voice recordings, files, screenshots, etc.? types we render

**Comment by amyblais** [comment]
Hi @JeffSchering, this is ready to be reviewed.

File: source/help/messaging/attaching-files.rst [hunk 2]
```diff
@@ -3,62 +3,77 @@ Attaching Files
...
 
 Downloading Files
 ~~~~~~~~~~~~~~~~~
-Download an attached file by clicking the download icon next to the file
-thumbnail or by opening the file previewer and clicking **Download**.
+You can download an attached file in two ways:
+
+-  click the download icon next to the file thumbnail
+-  open the file previewer and click **Download**
+
```
Comment (JeffSchering) [inline source/help/messaging/attaching-files.rst]: In this case, it's probably better to just mention the first method only. It's the simplest (requires only one click) and people can discover the other method easily if they open the file previewer. Plus it's already covered by the File Previewer section of this doc.

File: source/help/messaging/attaching-files.rst [hunk 3]
```diff
@@ -3,62 +3,77 @@ Attaching Files
...
+Viewing Media:
 ~~~~~~~~~~~~~~~~~~~~~
-If you are trying to preview a media type that is not supported, the
-file previewer will open a standard media attachment icon. Supported
-media formats depend heavily on your browser and operating system, but
-the following formats are supported by Mattermost on most browsers:
+If you are trying to preview a media type that is not supported, the 
+file previewer will open a standard media attachment icon. Supported 
+media formats depend on your browser and operating system, but the 
+following formats are supported on most browsers:
```
Comment (JeffSchering) [inline source/help/messaging/attaching-files.rst]: I think the same comment I made about the *How to Attach Files* section above applies here. These items can be compressed down to one or maybe two sentences each in the bullet list.

File: source/help/messaging/attaching-files.rst [hunk 4]
```diff
@@ -3,62 +3,77 @@ Attaching Files
...
-Supported Media Types
+Viewing Media:
 ~~~~~~~~~~~~~~~~~~~~~
-If you are trying to preview a media type that is not supported, the
-file previewer will open a standard media attachment icon. Supported
-media formats depend heavily on your browser and operating system, but
-the following formats are supported by Mattermost on most browsers:
+If you are trying to preview a media type that is not supported, the 
+file previewer will open a standard media attachment icon. Supported 
+media formats depend on your browser and operating system, but the
```
Comment (JeffSchering) [inline source/help/messaging/attaching-files.rst]: I'm not sure I understand what is meant by "open a standard media attachment icon"

File: source/help/messaging/attaching-files.rst (hunk 3 above)
Comment (amyblais) [inline source/help/messaging/attaching-files.rst]: These tips are very helpful, thank you for taking the time to read the doc in detail! I started to make revisions and will continue to tweak the doc and to add screenshots.

File: source/help/messaging/attaching-files.rst [hunk 5]
```diff
@@ -1,64 +1,50 @@
...
+Click the thumbnail of an attached file to open it in the file previewer.
 
 Downloading Files
 ~~~~~~~~~~~~~~~~~
-Download an attached file by clicking the download icon next to the file
-thumbnail or by opening the file previewer and clicking **Download**.
+You can download an attached file by:
+
+-  Clicking the download icon next to the file thumbnail
+
```
Comment (JeffSchering) [inline source/help/messaging/attaching-files.rst]: A list should have two or more items. If only one item, then it's not a list. This should be:

`You can download an attached file by clicking the download icon next to the file thumbnail.`

File: source/help/messaging/attaching-files.rst [hunk 6]
```diff
@@ -1,64 +1,50 @@
...
+Sharing Public Links
+~~~~~~~~~~~~~~~~~~~~
+Public links allow you to share attachments with people outside your 
+Mattermost team.
+
+-  Click the thumbnail of an attachment > click **Get Public Link**
+
```
Comment (JeffSchering) [inline source/help/messaging/attaching-files.rst]: This should be something like 

Public links allow you to share attachments with people outside your Mattermost team. To share an attachment, click the thumbnail of an attachment then click **Get Public Link**.

And I think it's not just outside the Team, but with anyone on the Internet. (A Mattermost instance can have several teams) so you should rewrite it a bit to clarify that.

File: source/help/messaging/attaching-files.rst [hunk 7]
```diff
@@ -1,64 +1,50 @@
...
+.. Tip::
+If **Get Public Link** is not visible in the file previewer
+and you prefer that the feature is enabled, you can request your System 
+Administrator to enable the feature from the System Console under
```
Comment (JeffSchering) [inline source/help/messaging/attaching-files.rst]: System Administrator > System Admin

File: source/help/messaging/attaching-files.rst [hunk 8]
```diff
@@ -67,8 +53,10 @@ the following formats are supported by Mattermost on most browsers:
...
-File Size Limitations
+Attachment Limits and Sizes
 ---------------------
-Mattermost supports a maximum of five attached files per post.
+Please note that Mattermost supports a maximum of five (5) attached 
+files per post.
 
-The default maximum file size is 50 MB, but this can be changed by the System Adminstrator.
+The default maximum file size is 50 MB, but this can be changed by the 
+System Administrator.
```
Comment (JeffSchering) [inline source/help/messaging/attaching-files.rst]: System Administrator > System Admin

File: source/help/messaging/attaching-files.rst [hunk 9]
```diff
@@ -25,22 +25,20 @@ Click the thumbnail of an attached file to open it in the file previewer.
...
+You can download an attached file by clicking the download icon next to the file 
+thumbnail.
 
 Sharing Public Links
 ~~~~~~~~~~~~~~~~~~~~
-Public links allow you to share attachments with people outside your 
-Mattermost team.
-
--  Click the thumbnail of an attachment > click **Get Public Link**
+Public URL's allow you to share attachments with anyone outside the Mattermost
```
Comment (JeffSchering) [inline source/help/messaging/attaching-files.rst]: URL's > URLs

Comment (amyblais) [inline source/help/messaging/attaching-files.rst]: Thank you for all the review you have done on this document, the comments you have made have been really helpful!

**Comment by jasonblais** [comment]
@amyblais For updating "Attaching Files" and "Sending Messages" help doc PRs, they're a significant improvement to our existing ones, thank you! (and thanks to Jeff as well!)

Wondering if you had a list of best practices that you prepared while working on the docs and if we can apply them for other help docs?

One question that comes to mind is screenshots -- should we add any the these two pages? Why or why not? ([An article about screenshots that might be useful](https://opensource.com/business/15/9/when-does-your-documentation-need-screenshots))

**Comment by amyblais** [comment]
@jasonblais I have prepared a list of best practices for end user docs and I am currently thinking about where to add them. I had originally thought of putting them in "Documentation Guidelines" but it has more general guidelines for all documentation.

I did not add any screenshots to these documents because they are generally quite straightforward, but I can think about this further.

**Comment by jasonblais** [comment]
@amyblais Sounds good.

 I think it might be okay to open a WIP PR and we can figure out where we can put it.

**Pull request merged**